Snowflake.Data/Core/FileTransfer/SFFileTransferAgent.cs

```diff
@@ -732,7 +732,8 @@ private ISFRemoteStorageClient renewExpiredClient()
             }
             catch (Exception ex)
             {
-                throw ex;
+                Console.WriteLine("Unhandled exception while uploading file.", ex);
```

Comment:
Shouldn't this be writing to `Logger`, not `Console`?

Reply:
It most definitely should! Let me fix that real fast, thanks!